content/influxdb3/cloud-dedicated/admin/tables/create.md

```diff
@@ -161,21 +167,26 @@ If a table doesn't have a custom partition template, it inherits the database's
    can sort on column headers or use the **Search** field to find a specific cluster.
 4. In the database list, find and click the database you want to create a table in. You
    can sort on column headers or use the **Search** field to find a specific database.
-4. Click the **New Table** button above the table list. 
+5. Click the **New Table** button above the table list. 
    The **Create table** dialog displays.
-    <img src="/img/influxdb3/cloud-dedicated-admin-ui-create-table.png" alt="Create table dialog" />
-5. In the **Create table** dialog, provide a **Table name**.
-6. Make sure the **Use default partitioning** toggle is set to **Off**
-7. Provide the following:
-
-   - **Custom partition template time format**: The time part for partitioning data.
-   - _Optional_: **Custom partition template tag parts**: The tag parts for partitioning data.
-   - _Optional_: **Custom partition template tag bucket parts**: The tag bucket parts for partitioning data.
-8. _Optional_: To add more parts to the partition template, click the **Add Tag** button.
-9. Click the **Create Table** button to create the table.
+
+   {{< img-hd src="/img/influxdb3/cloud-dedicated-admin-ui-create-table-default.png" alt="Create table dialog" />}}
+
+6. In the **Create table** dialog, provide a **Table name**.
+7. Toggle **Use custom partitioning** to **On**.
+   The **Custom partition template** section displays.
+
+   {{< img-hd src="/img/influxdb3/cloud-dedicated-admin-ui-create-table-custom-partitioning.png" alt="Create table dialog with custom partitioning" />}}
+
+8. Provide the following:
+
+   - **Custom partition template time format**: The time part for partitioning data (yearly, monthly, or daily).
+   - _Optional_: **Custom partition template tag parts**: The [tag parts](/influxdb3/cloud-dedicated/admin/custom-partitions/partition-templates/#tag-part-templates) for partitioning data.
+   - _Optional_: **Custom partition template tag bucket parts**: The [tag bucket parts](/influxdb3/cloud-dedicated/admin/custom-partitions/partition-templates/#tag-bucket-part-templates) for partitioning data.
+9. _Optional_: To add more parts to the partition template, click the **Add Tag** button. For more information, see [Partition template requirements and guidelines](#partition-template-requirements-and-guidelines).
```

Comment:
9 Looks like it may be duplicative of 8 now, since they need to click "Add tag" to get to the menu for adding template tag parts or tag bucket parts.